docs/tutorials/timeseries/forecasting-faq.md

```diff
@@ -1,42 +1,40 @@
 # AutoGluon Time Series FAQ
 
+## What forecasting tasks can AutoGluon be used for?
+AutoGluon can generate **probabilistic** multi-step-ahead forecasts for one or multiple **univariate** time series.
+For example, you can use AutoGluon to forecast daily sales of multiple products over the next month.
 
-## Where can I find more information about the models/metrics?
+AutoGluon also supports additional information, such as time-independent static features (e.g., location of the store)
+and time-dependent covariates (e.g., price of the product each day).
+See the [In Depth Tutorial](forecasting-indepth.ipynb) for more details.
 
-Metrics are implemented in the `autogluon.timeseries.evaluator` module. We also follow some of
-the same conventions followed by GluonTS in their evaluation.
-Please refer to
-the GluonTS [documentation](https://ts.gluon.ai/stable/api/gluonts/gluonts.html) and
-[github](https://github.com/awslabs/gluon-ts) for further information.
+Currently, AutoGluon does not support features such as hierarchical forecasting and forecast explainability, but we will consider adding them in the future.
 
-A detailed description of evaluation metrics is also available at
-[here](https://docs.aws.amazon.com/forecast/latest/dg/metrics.html).
+## How can I get the most accurate forecasts?
+To maximize the forecast accuracy, set the `predictor.fit()` argument `presets="best_quality"` or `presets="high_quality"` and provide a high `time_limit`.
 
-## How can I get the most accurate forecast predictions?
+## How should I choose the evaluation metric?
+See ["How to choose and interpret the evaluation metric?" in the In Depth Tutorial](forecasting-indepth.ipynb)
 
-Generally setting the `predictor.fit()` argument `presets="best_quality"` or `presets="high_quality"` will result in high accuracy.
-Alternative options include manually specifying hyperparameter search spaces for certain models and
-manually increasing the number of hyperparameter optimization trials.
+## Are there any restrictions on the data that I can pass to TimeSeriesPredictor?
+See ["What data format is expected by `TimeSeriesPredictor`?" in the In Depth Tutorial](forecasting-indepth.ipynb)
 
 
 ## Can I use GPUs for model training?
 
-Yes! Most of the deep learning models used by `autogluon.timeseries` support GPU training.
-PyTorch models will have GPU enabled by default. If you also want to use MXNet models, make sure you have installed CUDA and the GPU version of MXNet.
+Yes! All deep learning models used by `autogluon.timeseries` support GPU training.
```

Comment:
Can we add some instructions to help users understand if GPU is used/supported for TS training? For example, specific log entry emitted

Reply:
Currently this would only be visible for `verbosity=4` (in the pytorch-lightning device information), I don't think that's recommended. We could add the following log entry inside `TimeSeriesPredictor.fit`
```python
if torch.cuda.is_available():
    logger.info("GPU detected. Models that support GPU training will be trained on GPU")
```
Are you in favor of doing this?

Reply:
Do we have log entry from lightning such as:

```
Using 16bit None Automatic Mixed Precision (AMP)
GPU available: True (cuda), used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
HPU available: False, using: 0 HPUs
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]
```

Reply:
These only appear at `verbosity=4` together with the rest of the GluonTS output (which can be very crowded). I wouldn't recommend that users set `verbosity=4` unless debugging.